pip-dep/requirements_dev.txt

```diff
@@ -6,7 +6,9 @@ jupyter>=1.0.0
 pre-commit>=1.16.1
 pyopenssl>=19.0.0
 pyshark>=0.4.2.4
+pytest-asyncio
```

Comment:
Q: Should we add versions to this?
I got a warning for this and for the scipy I got an error saying that I did not have it on my system.
Q2: Is this a good place for them?